stock_inventory_lockdown/models/stock_location.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
+
+
+class StockLocation(models.Model):
+    """Refuse changes during exhaustive Inventories"""
+    _inherit = 'stock.location'
+    _order = 'name'
+
+    @api.multi
+    @api.constrains('location_id')
```

Comment:
This is also not working as it worked before, raises on creation but not on write...

Reply:
Weird as the trigger is location_id.

The code is different from the initial migration commit.

Here you don't take into account self in recordset to check.

Reply:
But if we change the _check_inventory_location_id to check the self, then the create won't work correctly :S